sql/updates/world/3.3.5/9999_99_99_world.sql

```diff
@@ -0,0 +1,2 @@
+-- Show if player is online in lookup player commands
+UPDATE `trinity_string` SET `content_default` = '  %s (GUID %u) |cff00ff00%s|r' WHERE (`entry` = '329');
```

Comment:
I'm curious, whats the output if the player is offline?

Reply:
I have used the same behavior that the "(Offline)" in the .pinfo command but in this case, it shows the opposite (If player is online).
In the case of .pinf command
```cpp
handler->PSendSysMessage(LANG_PINFO_PLAYER, target ? "" : handler->GetTrinityString(LANG_OFFLINE).....
```
In this case:
```cpp
`handler->PSendSysMessage(LANG_LOOKUP_PLAYER_CHARACTER, name.c_str(), guid, online ? "Online" : "");
```
So, if player is offline, output is "".
Once this shows which player is online, I do not see any reason to show something if player is offline. It would look redundant.

Reply:
Hmm, I take back my approval, make that "Online" string possible to localize (use trinity_string)

Reply:
My questions was actually oriented into what happens to the trinity_string if the given parameter is empty, as in the charater is offline. Server sends the color "pattern" with empty content?

Reply:
I will remove the color when I move it to trinity_string
Is it better?